src/en/alternativescans/src/eu/kanade/tachiyomi/extension/en/alternativescans/AlternativeScans.kt

```diff
@@ -0,0 +1,176 @@
+        return MangasPage(mangas, false)
+    }
+
+    // Latest
+    override fun latestUpdatesRequest(page: Int): Request = popularMangaRequest(page)
+
+    override fun latestUpdatesParse(response: Response): MangasPage = popularMangaParse(response)
+
+    // Search
+    override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> = Observable.error(UnsupportedOperationException("Search not yet implemented."))
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request = throw UnsupportedOperationException("Search not yet implemented.")
+    override fun searchMangaParse(response: Response): MangasPage = throw UnsupportedOperationException("Search not yet implemented.")
+
+    // Details
+    override fun mangaDetailsRequest(manga: SManga): Request {
+        val nick = manga.url.substringAfter("/manga/").substringBefore("?")
+        val seriesId = manga.url.substringAfter("?id=")
+        return GET("$apiUrl/api/admin/getSeriesDetails/$seriesId/$nick", apiHeaders)
+    }
+
+    override fun mangaDetailsParse(response: Response): SManga {
+        val result = json.decodeFromString<MangaDetailsResponse>(response.body.string())
+        return SManga.create().apply {
+            title = result.seriesDetails.title
+            description = result.seriesDetails.desc
+            author = result.seriesDetails.author
+            artist = result.seriesDetails.artist
+            genre = result.seriesDetails.genre
+            status = parseStatus(result.seriesDetails.manga_status)
+            thumbnail_url = result.seriesDetails.thumbnail
+        }
+    }
+
+    private fun parseStatus(status: String?): Int {
+        return when (status?.lowercase()) {
+            "releasing" -> SManga.ONGOING
+            "completed" -> SManga.COMPLETED
+            else -> SManga.UNKNOWN
+        }
+    }
+
+    // Chapters
+    override fun chapterListRequest(manga: SManga): Request {
+        val nick = manga.url.substringAfter("/manga/").substringBefore("?")
+        val seriesId = manga.url.substringAfter("?id=")
+        return GET("$apiUrl/api/admin/getSeriesDetails/$seriesId/$nick", apiHeaders)
+    }
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val result = json.decodeFromString<MangaDetailsResponse>(response.body.string())
+        val nick = response.request.url.pathSegments.last()
+        val seriesId = response.request.url.pathSegments[response.request.url.pathSegments.size - 2]
+        val dateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", Locale.US)
+        return result.releases.map {
+            SChapter.create().apply {
+                name = "Chapter ${it.chapterNo}" + (it.chapterName?.let { ": $it" } ?: "")
+                url = "/reader.html?id=$seriesId&series=$nick&chapter=${it.chapterNo}"
+                date_upload = try { dateFormat.parse(it.uploadDate)?.time ?: 0L } catch (e: Exception) { 0L }
```

Comment:
```suggestion
                date_upload = dateFormat.tryParse(it.uploadDate)
```
from `keiyoushi.utils.tryParse`